core/src/main/scala/chisel3/internal/package.scala

```diff
@@ -63,6 +63,16 @@ package object internal {
     }
   }
 
+  // Sanitizes a name, e.g. from a `HasId`, by stripping all non ANSI-C characters
+  private[chisel3] def sanitize(s: String, leadingDigitOk: Boolean = false): String = {
+    // TODO what character set does FIRRTL truly support? using ANSI C for now
+    def legalStart(c: Char) = (c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z') || c == '_'
+    def legal(c:      Char) = legalStart(c) || (c >= '1' && c <= '9')
```

Comment:
```suggestion
    def legal(c:      Char) = legalStart(c) || (c >= '0' && c <= '9')
```
Very subtle copy-paste mistake 😂 Seems like an accidental `ctrl-a` in vim.